src/editor/Gui.cpp

```diff
@@ -915,6 +1124,9 @@ void Gui::drawMenuBar() {
 			showLightmapEditorWidget = !showLightmapEditorWidget;
 			showLightmapEditorUpdate = true;
 		}
+		if (ImGui::MenuItem("Map merge", "", showMergeMapWidget)) {
+			showMergeMapWidget = !showMergeMapWidget;
+		}

```

Comment:
leave out for now

Reply:
I remove "model merge option" and now it works as in command line.

Reply:
Not needed since there is no way to open multiple maps.